website/pages/docs/how-to-guides/how-to-load-infrastructure-data-into-athena.md

```diff
@@ -0,0 +1,280 @@
+
+```bash copy
+export BUCKET_NAME=cloudquery-athena-example
+aws s3 mb s3://$BUCKET_NAME
+```
+
+Now we can upload the files to the bucket. Athena requires every table to be in its own directory, so we'll use a little bash scripting to upload each CSV file into its own directory:
+
+```bash copy
+for file in cq_csv_output/*.csv; do
+    filename=$(basename "$file")
+    foldername="${filename%.*}"
+    aws s3 cp "$file" "s3://$BUCKET_NAME/$foldername/$filename"
+done
+```
+
+You should now see a large number of objects in the bucket, which we can verify by listing the items:
+
+```bash copy
+aws s3 ls s3://$BUCKET_NAME
+```
+
+### 6. Create a Glue Crawler
+
+Athena can query data in S3, but it needs to know the schema of the data in order to do so. We can use a Glue Crawler to automatically infer the schema of the data and create a table in the Athena database we created in the previous step. We'll use the AWS CLI again.
+
+First we'll create a role for the crawler to use. It will need access to the S3 bucket we created in the previous step. The following trust policy will give the role what it needs:
+
+```json copy title="crawler-trust-policy.json"
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "glue.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+```
+
+With the above policy saved as `crawler-trust-policy.json`, we can now create the role (make a note of the ARN for the newly created role):
+
+```bash copy
+aws iam create-role \
+    --role-name cloudquery-athena-example-crawler \
+    --assume-role-policy-document file://crawler-trust-policy.json
+```
+
+We should also attach a policy to the role that gives it access to the S3 bucket we created in the previous step. The following policy will give the crawler access to the S3 bucket. Make sure to update the bucket name to the value of `BUCKET_NAME` you chose earlier (in our example, `cloudquery-athena-example`):
+
+```json copy title="crawler-policy-s3-access.json"
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
+                "s3:PutObject"
+            ],
+            "Resource": [
+                "arn:aws:s3:::cloudquery-athena-example*"
+            ]
+        }
+    ]
+}
+```
+
+Let's attach this policy to the role:
+
+```bash
+aws iam put-role-policy \
+    --role-name cloudquery-athena-example-crawler \
+    --policy-name cloudquery-athena-example-crawler-s3-access \
+    --policy-document file://crawler-policy-s3-access.json
+```
+
+The crawler will also need additional permissions to perform all its tasks, such as writing CloudWatch logs. We attach another policy to the role to give it these permissions. You should review these permissions to ensure they are appropriate for your use case:
```

Comment:
`put-role-policy` attaches an inline policy to the IAM role, we can only have 1 inline policy at a time - so this may overwrite the previous command.  Should we combine these policy sections?

Reply:
We can combine them, but it seems to work fine to have two (This is how I set it up while writing this guide):

![Screenshot 2022-12-23 at 15 38 51](https://user-images.githubusercontent.com/1121616/209361731-3c91aec8-7201-44e9-89f6-3cbde1466112.png)

Reply:
You're right about multiple inline policies.  I was incorrect about only having 1.  They just count towards the same quota.  

It's good like you have it!